### DISCUSSION TITLE
Pre-trained/Fine tuning for

### DISCUSSION THREAD

**Discussion by ggropican** [Project-MONAI/tutorials#664] [Q&A]
Hello Monai Developers!

I am currently trying to train a stack of 2D images(329 images) using 'unet_training_array.py' from the tutorial provided, however the training loss still high. So I am planning to use a pre-trained model instead but couldn't find any tutorial from Monai. Do you have any advice or tutorial for using a pre-trained model?

My image size is [1056,1056]
Thank you very much in advance

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

Could you please help share some comments here?

Thanks in advance.

**Comment by yiheng-wang-nv** [comment]
Hi @ggropican , monai does not provide pretrained weights for `UNet`. As for your task, could you please provide some more information, such as 1) what transforms do you used? 2) what are the training and validation losses for each epoch?
I may need more information to provide more detailed advices, thanks!

**Reply by ggropican** [reply]
Hello @yiheng-wang-nv , The raw images are size of [2088,1984] and there are 140 images. I am operating at batch size of 21, learning rate of 1e-4 and these are my current transforms: 
train_imtrans = Compose(
    [
        LoadImage(image_only=True),
        AddChannel(),
        Resize(spatial_size=(1052,1000),mode='bilinear'),
        SpatialPad((1056,1056),"symmetric"),
        ScaleIntensity(0,1),
        EnsureType(),
    ]
)
epoch1
training loss: 0.9980
epoch2
training loss: 0.9976
val_meandice: 0.3238
epoch3
training loss: 0.9974
epoch4
training loss: 0.9872
val_meandice: 0.2547
epoch5
training loss: 0.9804
epoch6
training loss: 0.9624
val_meandice: 0.1399
epoch7
training loss: 0.9383
epoch8
training loss: 0.9283
val_meandice: 0.0635
epoch9
training loss: 0.9206
epoch10
training loss: 0.9256
val_meandice: 0.0298
epoch11
training loss: 0.9202
epoch12
training loss: 0.9356
val_meandice: 0.01425
<img width="380" alt="Screen Shot 2022-04-22 at 14 51 06" src="https://user-images.githubusercontent.com/104089968/164727909-64ba2303-2148-4cfd-a276-8a1d8cdaf153.png">

(I got hold of a pre-trained UNET model on a pth file, do you have any recommendations on how to load it?)
Thank you very much!
